src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -773,31 +768,45 @@ func (c controller) waitForOLMOperators() bool {
 		return true
 	}
 	for _, operator := range operators {
-		csvName, err := c.kc.GetCSVFromSubscription(operator.Namespace, operator.SubscriptionName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get subscription of operator %s", operator.Name)
-			continue
-		}
 
-		csv, err := c.kc.GetCSV(operator.Namespace, csvName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get %s", operator.Name)
-			continue
-		}
+		//FIXME: We check the SC of the OCS operator WA bug 1968606
+		//Remove this code when bug 1968606 is fixed
+		if operator.Name == "ocs" {
+			//get SC
+			sc, err := c.kc.GetSC()
```

Comment:
I just wonder whether maybe we don't want to check both SC and CSV for OCS? Or SC is enough?

Reply:
SC should be enough

Reply:
Well SC could be also `StorageClass` for example, which may be confusing.